### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by DomAyre**
Adds a simple demo which spins up a KMS and all 4 B&A services, and then allows the user to send example requests

### CODE REVIEW

File: .devcontainer/scripts/generate_system_bazelrc
Comment (takuro-sato) [inline .devcontainer/scripts/generate_system_bazelrc:1]: Should we clarify copyright for these Google's file?
Or maybe we can explicitly copy them from the B&A repo in a script

Comment (DomAyre) [reply .devcontainer/scripts/generate_system_bazelrc:1]: We could copy them from the B&A repo, but then google can break our codespace underneath us.

I think leaving the files untouched is safer, I've mentioned them in the NOTICE file too to be safe but don't have strong opinions

**Comment by beejones** [comment]
Just a reminder to add test_encryption.sh

**Comment by beejones** [comment]
No instructions how to run tests

**Comment by DomAyre** [comment]
> Just a reminder to add test_encryption.sh

I believe we only want to spin up the services and send requests for now

**Comment by beejones** [comment]
Please checkout the other repo's (demo) in the current directory or at the same level so developers don't have to switch to a complete different directory to see the code.
We need to install demo where the developer already decided where  he/she wants to checkout code.

File: .devcontainer/scripts/generate_system_bazelrc
Comment (takuro-sato) [reply .devcontainer/scripts/generate_system_bazelrc:1]: I didn't realize NOTICE. Thanks for clarifying

**Comment by DomAyre** [comment]
As discussed, we're happy to merge without changing the default DEMO_WORKSPACE

We also want to keep this repo simple so we won't run tests here

**Pull request merged**